src/main/java/org/hydev/mcpm/client/Downloader.java

```diff
@@ -125,22 +122,4 @@ public Downloader threads(int threads)
         this.threads = threads;
         return this;
     }
-
-    /**
-     * Displays a demo for.
-     *
-     * @param args Arguments are ignored.
-     */
-    public static void main(String[] args) throws IOException {
-        // Remember to chang link to test
-        String link = "https://sd.blackball.lv/library/Introduction_to_Algorithms_Third_Edition_(2009).pdf";
-        File out = new File("./Introduction_to_Algorithms_Third_Edition.pdf");
-        String link1 = "https://www.iusb.edu/students/academic-success-programs/academic-centers-for-excellence/docs/Basic%20Math%20Review%20Card.pdf";
-        File out1 = new File("./Math.pdf");
-        Downloader downloader = new Downloader();
-        Map<String, File> urls = new HashMap<>();
-        urls.put(link, out);
-        urls.put(link1, out1);
-        downloader.downloadFiles(urls);
```

Comment:
?

Reply:
I just moved the testing in main to its own test class